apps/src/lib/kits/maker/portScanning.js

```diff
@@ -57,7 +57,7 @@ export function ensureAppInstalled() {
  */
 function listSerialDevices() {
   return new Promise((resolve, reject) => {
-    ChromeSerialPort.list((error, list) => error ? reject(error) : resolve(list));
+    SerialPort.list((error, list) => error ? reject(error) : resolve(list));
```

Comment:
Supporting both the Chrome connector app and the Electron bridge at this layer will be as simple as `(SerialPort || ChromeSerialPort).list(...);` - most of the complexity actually lies in showing sensible descriptions and intermediate steps on the setup page according to the user's current setup.